CenterForOpenScience/osf.io
# Feature/Requirements Upgrade [OSF-8936]

## Ticket
https://openscience.atlassian.net/browse/OSF-8936

(Please see ticket for notes)

## QA
All user-entered fields for preprint providers need to be thoroughly checked - preprint footers, for example. Make sure the formatting doesn't look weird.
(Note to self: add more detail to this section)
